_guide/_pages/interviews/technical.md

```diff
@@ -61,89 +61,29 @@ to produce and defend a desired solution. Preface by saying there’s no right a
 
 Help guide the candidate to the right line of thinking, don’t allow them to get too bogged down in the wrong line of thinking — help them get back on track.
 
-* ["How would you design software to run elevators? Can you talk through a high-level design? How would accommodating different building types change your design?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.tgd47iei6k4w)
+* ["Let's say you and your team are tasked to build a feature or an app that would support photo sharing. How would you approach the design and build for this?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.tgd47iei6k4w)
 
-* ["Tell me about a difficult technical problem that you helped solve: what was the problem, and how was it solved?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.ory7bezha9al)
+* ["Tell me about a difficult technical problem that you helped solve: what was the problem, and how was it solved?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.ory7bezha9al)
+
+* ["As much as we like to rely on well-trod open source paths, engineers often encounter technical stacks that they have little or no familiarity with. What is your approach to tackling new or unknown software technologies?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.x5ub9jdkb2nu)
 
 ### Agile orientation
 
-* ["Tell me about your experience with agile or other collaborative working styles. How does working in an agile way affect the technical choices you make?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.p3zgydisjuv3)
+* ["Whether or not you’ve had direct experience with agile, in an agile environment, how do you approach trade-offs?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.aq4p1thowhq)
```

Comment:
I’m not sure I know what this question means — it could be a "discuss the process" question or a "tell me a time you balanced" question or a "how do you compare apples to oranges" question. 

In my mind, the previous question asked specifically about how iteration can be applied to breaking apart technical work. If I recall, it wasn’t a particularly insightful question, so I’m glad we’re targeting it for revision!

Reply:
Fair point! IIRC we struggled with the wording here. I've yet to hear a good answer to the question in its current form, so was hoping to make it more explicitly about how you make decisions in a process where getting everything perfect isn't an option.

Now that I'm looking at it again, I wonder if we should go with something along the lines of "How do you decide which work to prioritize in an agile or collaborative environment?" cc @lauraGgit

Reply:
Real-world question: How would you introduce agile concepts to a partner that only has experience with waterfall development (goals of question: to learn how they coach, to catch any superiority complex, and to gauge their understanding o agile concepts)

Also, for this question, maybe scrape off some sugarcoating and get down to brass tacks (with apologies for some horribly mixed metaphors): 
> How do you handle conflicting opinions about what should be in a given sprint?